### PR TITLE
Minor Gradle tweaks

### PR DESCRIPTION
**Pull request by Goooler**
Kotlin sources in idea-plugin module can't be viewed in IDEA as we removed its dependency explicitly. Now we `compileOnly` stdlib for all modules that applied Kotlin plugin, no need to worry small things like this.

### CODE REVIEW

**Comment by Goooler** [comment]
Will squash this PR.

File: idea-plugin/build.gradle.kts [hunk 1]
```diff
@@ -62,7 +59,7 @@ compose.resources {
 }
 
 composeCompiler {
-    stabilityConfigurationFile = rootProject.layout.projectDirectory.file("stability_config.conf")
+    stabilityConfigurationFile = layout.projectDirectory.file("stability_config.conf")
```
Comment (Goooler) [inline idea-plugin/build.gradle.kts:62]: Should we share this config for App and Plugin?

File: idea-plugin/build.gradle.kts [hunk 2]
```diff
@@ -62,7 +59,8 @@ compose.resources {
 }
 
 composeCompiler {
-    stabilityConfigurationFile = rootProject.layout.projectDirectory.file("stability_config.conf")
+    enableStrongSkippingMode = true
+    stabilityConfigurationFile = layout.projectDirectory.file("stability_config.conf")
```
Comment (Goooler) [inline idea-plugin/build.gradle.kts:63]: Should we share this config and enable strong-skip for App and Plugin?

Comment (Goooler) [reply idea-plugin/build.gradle.kts]: 19053bbde6b0ec2186596cbf7ec1177bb62bb92f

Comment (egorikftp) [reply idea-plugin/build.gradle.kts]: will be removed after release kotlin 2.0.20 
https://kotlinlang.org/docs/whatsnew-eap.html#strongskipping-mode-enabled-by-default

**Review by egorikftp — APPROVED** [review]

File: idea-plugin/build.gradle.kts (hunk 2 above)
Comment (Goooler) [reply idea-plugin/build.gradle.kts]: Nice, will remove it then.

**Pull request merged**